changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## unreleased
+* Small styling improvement
+* Option `showSubmit` for contact forms is now named `standalone`. The old option will remain as alias until the next major version
```

Comment:
How will we communicate this deprecation to downstream developers?

Reply:
I will add it to `upgrading.md`